db/seeds.rb

```diff
@@ -11,7 +11,7 @@
 Species.where(name: 'Nubian ibex').first_or_create
 Species.where(name: 'Scimitar-horned Oryx').first_or_create
 Species.where(name: 'Sable Antelope').first_or_create
-Species.where(name: 'Tajik Markhor').first_or_create
+Species.where(name: 'Tadjik Markhor').first_or_create
```

Comment:
It seems we use the single_quote style. Maybe we switch from double_quote to single_quote instead and see if that makes less noise before disabling?